src/android.rs

```diff
@@ -62,6 +62,9 @@ impl Device for AndroidDevice {
         let bundle_path = exe.parent().ok_or("no parent")?.join("dinghy");
         let bundled_exe_path = bundle_path.join(exe_file_name);
 
+        debug!("Removing previous bundle {:?}", bundle_path);
+        fs::remove_dir_all(&bundle_path).ok();
```

Comment:
use `let _ = fs::remove_dir_all(&bundle_path)` as it's how it's done in other parts of the code